src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private Random random = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public FigureDrawable getRandomFigure() {
```

Comment:
The method `getRandomFigure()` should not return `null`. In the `switch` case, you can handle the default case by returning a specific figure instead of having a separate method to handle the default figure. This avoids the need for additional method calls and simplifies the control flow.